src/styles/main.css

```diff
@@ -88,8 +90,6 @@ main {
   padding-right: 32px;
 
   background-color: #d3d3d3;
-
-  animation: slide-in 500ms ease-in;
```

Comment:
Did you want to remove the animations?

Reply:
Yup! I did. 
On mobile it was pretty jarring- My branch has kind of turned into a bit of a more holistic makeover 😅
I'm going to be pushing some things up either tonight or tomorrow that redo the nav, add a footer, and cleanup the general page templates.